pyglider/utils.py

```diff
@@ -197,11 +197,10 @@ def get_profiles_new(ds, min_dp = 10.0, inversion=3., filt_time=100,
 
 
 def get_derived_eos_raw(ds):
-    # GPCTD requires a scale factor of 10 for conductivity. Legato does not
-    if 'LEGATO' in ds.conductivity.source:
-        r = ds.conductivity / seawater.constants.c3515
-    else:
-        r = 10 * ds.conductivity / seawater.constants.c3515
+    # GPCTD and slocum ctd require a scale factor of 10 for conductivity. Legato does not
+    if 'GPCTD' in ds.conductivity.source or 'sci_water_cond' in ds.conductivity.source:
```

Comment:
I'd still rather this was somehow in the yaml.  i.e. `conductivity.units` and then it is S/m for seabird, and mS/cm for RBR and we convert that way?

Reply:
That would be a better way of implementing it. I've changed the logic now so it checks for units rather than sensor name. Should be more robust to new sensors